nixos/roles/lamp.nix

```diff
@@ -0,0 +1,157 @@
+{ config, pkgs, lib, ... }:
+let
+  role = config.flyingcircus.roles.lamp;
+
+in {
+  options = {
+    flyingcircus.roles.lamp.enable =
+      lib.mkEnableOption "Flying Circus LAMP stack";
+  };
+
+  config = lib.mkIf role.enable (
+    let
+      apacheConfFile = /etc/local/lamp/apache.conf;
+      phpiniConfFile = /etc/local/lamp/php.ini;
+
+      services.httpd.user = "s-vmonitor";
```

Comment:
This doesn't make sense here in the let. services.httpd.user is set to nobody later.

Reply:
good catch!